purchase_propagate_qty/readme/HISTORY.rst

```diff
@@ -0,0 +1,4 @@
+12.0.1.0.0 (2020-04-30)
```

Comment:
if you don't update this manually or add a towcrier compatible this file is useless